MANIFEST.in

```
@@ -1,3 +1,4 @@
 include LICENSE
 include README.rst
 include CHANGELOG.rst
+graft tests
```

```suggestion
graft tests
global-exclude *.py[co]
```

I unfortunately can't seem to push to your branch.